### ISSUE TITLE
'Group by Album' splits some albums to 'Group by Album & Artists'

### ISSUE THREAD

**Issue by Utini2000** [CDrummond/cantata#1424]
**Describe the bug**
In the library overview I have selected "group by Album". 
This usually works for most Albums but for some it will display the Album several times. 
The Album gets an entry for each Artist that is contained in the Album. 
The Album-Tag is the same on all the entries, only the Artist tag differs.
But I select and want to group by Album, not by Artist :S 
See screenshot:

![cantata](https://user-images.githubusercontent.com/14877663/50559158-63f40200-0cf4-11e9-9926-107635722870.png)

**To Reproduce**
1. Go to 'Library'
2. Click on the three vertical lines
3. Select 'Group by' -> 'Album'

**Expected behavior**
I should have no entry per Album and not several entries for one Album.

**Screenshots**
![example](https://user-images.githubusercontent.com/14877663/50559198-addce800-0cf4-11e9-8ab9-a45014da43bb.png)

**Version Info**
 - OS: Archlinux
 - Cantata Version: 2.3.2
 - MPD Version: 0.21.3
 - CMake options (other than default): ?

**Additional context**
Thanks in advance! :)

**Comment by Utini2000** [comment]
@CDrummond Why would I need to set albumartist? 

I want to group by Album, not by Albumartist. 
Also I can't just edit some of my media because some of them are web-releases that I am also re-seeding.

I have seen other reports and they basically said the same.
Want to group Album, Album tag is set, can't edit other tags because of e.g. torrents or massive library that would need editing.

So there is no way to group by "Album" tag?

**Comment by CDrummond** [comment]
No. You need proper tags. Multiple artists have "Greatest Hits" albums - you can't group an album simply by its name.

**Comment by Utini2000** [comment]
I don't get it.

I buy a CD, it has an Album name, it contains several different artists.
I group by Album name to get that CD in one complete package (including all the different artists in this album).

I had my entire library grouped with the same tags on foobar2000 and it worked fine?

**Comment by skielred** [comment]
I'm facing the same issue, the vast majority of the albums I listen to don't have a specific album artist.
They are mainly collaboration works or compilations.
In those case, the album artist field is (and **has to**) be empty. Setting "Various" or "Various Artists" in it is not a fix, it's adding to the problem. If there is no album artist the sementically correct thing to do is to leave this field empty.

In those cases, I'd love to be able to use cantata to either
- Only use the album name when grouping, or
- Use another tag coupled with the album name when grouping, such as *year* or even better, the *publisher* tag
